.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+
+| Q               | A
+| --------------- | ---
+| Laravel version | 5.1 / 5.2
+| PHP version     | 5.6.x
```

Comment:
we need the PATCH version too for BOTH php and laravel, otherwise they might as well just say, "laravel 5", lol